openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGResources.java

```diff
@@ -283,6 +296,16 @@ public void close() {
      */
     public int getBoosterNumFeatures() { return this.boosterNumFeatures; }
 
+    /**
+     * Returns the name of the features in the model.
```

Comment:
I'd suggest to change to `Returns the name of the features in the trained LightGBM boosting model model.`
This may help to understand why the `booster` on the name.

Reply:
It's only to make it more coherent with the other getters